mapboxgl/templates/choropleth_alt.html

```diff
@@ -0,0 +1,204 @@
+{% extends "main.html" %}
+
+<!-- update legend item key style -->
+{% block extra_css %}
+<style type='text/css'>
+    .legend div span { border-radius: 20%; }
+</style>
+{% endblock extra_css %}
+
+{% block javascript %}
+var legend = document.getElementById('legend');
+
+mapboxgl.accessToken = '{{ accessToken }}';
+
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: '{{ styleUrl }}', 
+    center: {{ center }},
+    zoom: {{ zoom }},
+    transformRequest: (url, resourceType)=> {
+        if ( url.slice(0,22) == 'https://api.mapbox.com' ) {
+            //Add Python Plugin identifier for Mapbox API traffic
+            return {
+                url: [url.slice(0, url.indexOf("?")+1), "pluginName=PythonMapboxgl&", url.slice(url.indexOf("?")+1)].join('')
+            }
+        }
+        else {
+            //Do not transform URL for non Mapbox GET requests
+            return {url: url}
+        }
+    }
+});
+
+var joinData = {{ joinData }};
```

Comment:
Since the number of stops we generate will probably be less than 10k or so, this won't be TOO slow.  Ideally, we would allow the user to load this data async from a local file HTTP request, just like they can for a normal `data` object as a geojson source.

Not critical to hit for this PR

Reply:
Yep I'd really like to get this in as well for consistency.  Is there a way to iterate through geojson added to the map as a source?

Reply:
@akacarlyann yes, for geojson sources only, you can call `map.getSource('data')._data` to get a copy of the source geojson data.

Reply:
However, the `join data` in this case will not be a geojson source, no?  The polygons will be in a vector tile source, and all the join data will contain is a JSON format set of `key: value` pairs, used to create the `match` expression for the style.

Reply:
In this case, are you suggesting that the __data__ param can be either geojson or json?  Won't that get confusing?